### ISSUE TITLE
Switch implements device_state_attributes

### ISSUE THREAD

**Issue by tonicb78** [rospogrigio/localtuya#674]
Hello, below a warning about a tuya switch:

Logger: homeassistant.helpers.entity
Source: helpers/entity.py:549
First occurred: 20:45:20 (1 occurrences)
Last logged: 20:45:20

Entity switch.luces_navidad_balcon (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.

Let me know if you need further info.

Thanks!

**Comment by creeve4** [comment]
Same after upgrading to Home Assistant 2021.12.5

`Logger: homeassistant.helpers.entity
Source: helpers/entity.py:549
First occurred: 9:49:33 PM (5 occurrences)
Last logged: 9:49:34 PM

Entity switch.cabinet_lights (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
Entity switch.fireplace (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
Entity switch.electric_blanket (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
Entity switch.bedroom_plug (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
Entity switch.desk_lights (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.`

**Comment by Efekent** [comment]
I got the same warning also after 2021.12.5

`Logger: homeassistant.helpers.entity
Source: helpers/entity.py:549
First occurred: 09:43:14 (3 occurrences)
Last logged: 09:43:14

Entity switch.thermostaat (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
Entity switch.kachel (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
Entity switch.swing (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.`

**Comment by 2Max2Pax** [comment]
Same here:
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:549
First occurred: 6:41:32 PM (4 occurrences)
Last logged: 6:41:32 PM

```
Entity switch.presa1 (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
Entity switch.presa2 (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
Entity switch.presa3 (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
Entity switch.usb (<class 'custom_components.localtuya.switch.LocaltuyaSwitch'>) implements device_state_attributes. Please report it to the custom component author.
```

**Comment by Smosia** [comment]
Another integration fixes that problem by Rename device_state_attributes -> extra_state_attributes. 
Maybe that issue would help. 

https://github.com/custom-components/sensor.yandex_maps/pull/20

**Comment by jeremysherriff** [comment]
Fixed in #663
